src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Interop/DotNetInvoker.ts

```diff
@@ -0,0 +1,103 @@
+﻿import { MethodOptions, DotnetMethodArgumentsList } from './MethodOptions';
+import { platform } from '../Environment';
+import { System_String } from '../Platform/Platform';
+import { getRegisteredFunction } from './RegisteredFunction';
+
+export function invokeDotNetMethod<T>(methodOptions: MethodOptions, ...args: any[]): (T | null) {
+    const method = platform.findMethod(
+        "Microsoft.AspNetCore.Blazor.Browser",
+        "Microsoft.AspNetCore.Blazor.Browser.Interop",
+        "JavaScriptInvoke",
+        "InvokeDotnetMethod");
```

Comment:
Sorry to be fussy about this, but for the `.ts` files could we stick to the "2-space indent" and "single-quotes" conventions? The 2-spaces thing is now in the `.editorconfig` so VS should honour that by default. As for the quotes it might be a pointless variation from C# conventions but let's keep consistency until/if we decide to change it globally. Hope that's OK!